kubespawner/reflector.py

```diff
@@ -218,9 +218,12 @@ async def _list_and_update(self):
         kwargs = dict(
             label_selector=self.label_selector,
             field_selector=self.field_selector,
+            resource_version=resource_version,
             _request_timeout=self.request_timeout,
             _preload_content=False,
         )
+        if resource_version:
```

Comment:
```suggestion
        if resource_version is not None:
```

Reply:
this is intentional so that it also does not match the empty string which I'm pretty sure also means unset